bin/postman-test/collections/security-service.postman_collection.json

```diff
@@ -160,16 +105,17 @@
 							}
 						],
 						"url": {
-							"raw": "https://localhost:8200/v1/sys/key-status",
+							"raw": "https://localhost:8200/v1/auth/token/lookup-self",
```

Comment:
key-status is a check on the vault encryption key.  What was intended was a check on the vault token validity.  Repeated numerous places.